python/pyosdp_data.c

```diff
@@ -363,7 +363,11 @@ static int pyosdp_make_struct_cmd_file_tx(struct osdp_cmd *p, PyObject *dict)
 }
 
 /* Dummies for commands that don't have any body */
-static int pyosdp_make_struct_cmd_dummy(struct osdp_cmd *cmd, PyObject *obj) { return 0; }
+static int pyosdp_make_struct_cmd_dummy(struct osdp_cmd *cmd, PyObject *obj) { 
```

Comment:
No, the issue is because we are setting `cmd->id` in the wrong place. The methods `pyosdp_make_struct_cmd` and `pyosdp_make_dict_cmd` should set it so the dummy can stay as it is now.